### PR TITLE
Feature/share articles

### PR DESCRIPTION
**Pull request by PaulSchroeder**

### CODE REVIEW

File: src/components/modals/CopyToClipboard.vue [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+<script>
+export default {
+  model: {
+    prop: 'article',
+  },
+  props: ['article'],
+  computed: {
+    widgetLink() {
+      const baseURL = (location.hostname === 'localhost' || location.hostname === '127.0.0.1') ?
```
Comment (theorm) [inline src/components/modals/CopyToClipboard.vue]: `baseUrl` should use `this.publicPath` (see https://cli.vuejs.org/config/#publicpath). So you don't need to use a ternary operator here, just `${window.location.origin}/${this.publicPath}`

File: src/components/modules/SearchResultsListItem.vue [hunk 2]
```diff
@@ -51,6 +51,21 @@
         <collection-add-to
           v-bind:item="article"
           v-bind:text="$t('add_to_collection')" />
+
+        <div v-if="article.issue.accessRights === 'OpenPublic'" class="shareArticleControl d-inline ml-1">
```
Comment (theorm) [inline src/components/modules/SearchResultsListItem.vue]: Would be great to add `OpenPublic` as a constant somewhere and use this constant in the template.

Comment (PaulSchroeder) [reply src/components/modules/SearchResultsListItem.vue]: changed this a bit, accessRights are now per **article**

File: src/components/modals/CopyToClipboard.vue (hunk 1 above)
Comment (PaulSchroeder) [reply src/components/modals/CopyToClipboard.vue]: added publicPath variable from `config`

**Pull request merged**